packages/firestore/src/typings/firestore.d.ts

```diff
@@ -579,6 +586,29 @@ declare namespace firestore {
     get(fieldPath: string | FieldPath): any;
   }
 
+  /**
+   * A `QueryDocumentSnapshot` contains data read from a document in your
+   * Firestore database as part of a query. The document is guaranteed to exist
+   * and its data can be extracted with `.data()` or `.get(<field>)` to get a
+   * specific field.
+   *
+   * A `QueryDocumentSnapshot` offers the same API surface as a
+   * `DocumentSnapshot`. As deleted documents are not returned from queries,
```

Comment:
I think "deleted documents are not returned from queries" may invite unnecessary confusion.  Perhaps "Since query results are all existing documents, ..." or even just omit the justification and say, "offers the same API surface as a `DocumentSnapshot` but its exists property ..."

Reply:
Fixed with the same texts that I used in Android.